.github/workflows/build.yml

```diff
@@ -24,3 +25,18 @@ jobs:
         if: startsWith(github.ref, 'refs/tags/')
         with:
           files: rpms/*.rpm
+      - name: Deploy to nexus
+        uses: Alfresco/alfresco-build-tools/.github/actions/maven-deploy-file@OPSEXP-1693-deploy-file
+        if: startsWith(github.ref_name, 'OPSEXP-1693')
+        with:
+          group-id: org.imagemagick
+          artifact-id: imagemagick-distribution
+          repository-url: https://nexus.alfresco.com/nexus/content/repositories/thirdparty/
+          version: ${{ steps.build.outputs.built-version }}-gha
```

Comment:
why do we flag this version with gha?

Reply:
to avoid with an excess of caution to overwriting existing nexus packages already present (uploaded manually) and somehow hint that those packages are built on gha

Reply:
Then shall we make it product agnotic? (like `-ci` or `alfresco-build`)?